### PR TITLE
[RELEASE]: v0.9.9

### PR DESCRIPTION
**Pull request by humzaiqbal**
## Description of proposed changes
Stamping a new release v0.9.9 for this issue https://github.com/snorkel-team/snorkel/pull/1690
## Related issue(s)

Fixes # (issue)

## Test plan
Existing tests should cover it
## Checklist

Need help on these? Just ask!

* [ ] I have read the **CONTRIBUTING** document.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.
* [ ] I have run `tox -e complex` and/or `tox -e spark` if appropriate.
* [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rjurney** [comment]
Excited to see this happening!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,13 +5,31 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 ### [Removed]
 
+## [0.9.9] - 2022-04-14
+### [Added]
+
+* PR #1690: Bumping numpy version for Mac M1 compat
+* PR #1696: Fix linting
+* PR #1694: Fix test loss
+* PR #1693: Update black version to fix build
+* PR #1671: Fixing flaky test
```
Comment (bhancock8) [inline CHANGELOG.md:22]: Nit: consistent imperative voice ("Fixing" -> "Fix")

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,13 +5,31 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 ### [Breaking Changes]
 ### [Added]
 ### [Changed]
 ### [Deprecated]
 ### [Removed]
 
+## [0.9.9] - 2022-04-14
+### [Added]
+
+* PR #1690: Bumping numpy version for Mac M1 compat
```
Comment (bhancock8) [inline CHANGELOG.md:18]: "Bump numpy version to X.X for Mac M1 compatibility"

File: test/labeling/test_convergence.py [hunk 3]
```diff
@@ -77,15 +78,27 @@ def test_labeling_convergence(self) -> None:
             + [get_positive_labeling_function(divisor) for divisor in range(2, 9)]
             + [get_negative_labeling_function(divisor) for divisor in range(2, 9)]
         )
+        logging.info("CREATED LABELING FUNCTIONS")
```
Comment (bhancock8) [inline test/labeling/test_convergence.py:81]: What report at both info and debug level?

File: test/labeling/test_convergence.py [hunk 4]
```diff
@@ -77,15 +78,27 @@ def test_labeling_convergence(self) -> None:
...
+        logging.debug("CREATED LABELING FUNCTIONS")
         applier = PandasLFApplier(labeling_functions)
-        L_train = applier.apply(self.df_train, progress_bar=False)
+        logging.info("APPLED LABEL FUNCTIONS")
+        logging.debug("APPLED LABEL FUNCTIONS")
+        L_train = applier.apply(self.df_train, progress_bar=True)
+        logging.info("CREATED L TRAIN")
```
Comment (bhancock8) [inline test/labeling/test_convergence.py:87]: Why do info & debug everywhere else but only info here?

File: .circleci/config.yml [hunk 5]
```diff
@@ -119,7 +136,10 @@ jobs:
       - setup_dependencies
       - run:
           name: "Run Tox"
-          command:  tox
+          no_output_timeout: 50m
```
Comment (bhancock8) [inline .circleci/config.yml:139]: Looks like tests were sometimes failing right on this line. Can that be bumped up to 60m?

File: .circleci/config.yml [hunk 6]
```diff
@@ -93,24 +97,37 @@ jobs:
           command:  tox
   Python38-Integration-Tests: 
     docker:
-      - image: cimg/python:3.8
+      - image: cimg/python:3.6
```
Comment (bhancock8) [inline .circleci/config.yml:100]: Why are we starting from 3.6 and building 3.8 instead of starting from 3.8?

**Comment by rjurney** [comment]
Scikit learn is really old in requirements.txt, which makes the package a little obsolete in terms of running it. Is this something that could go on this release?

File: test/labeling/test_convergence.py (hunk 3 above)
Comment (humzaiqbal) [reply test/labeling/test_convergence.py]: Removed

File: .circleci/config.yml (hunk 6 above)
Comment (humzaiqbal) [reply .circleci/config.yml]: This was something done while testing which is set back

**Review by bhancock8 — APPROVED** [review]

**Pull request merged**